content/en/docs/user-documentation/networking/firewall.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Firewall"
+weight: 20
+description: Firewall considerations for MicroShift
+card:
+  name: networking
+  weight: 20
+---
+
+MicroShift does not require a firewall to run, but it's recommended. In the case of
+using firewalld the following ports should be considered:
+
+| Port(s)     | Protocol(s) | Description                                                  |
+| ----------- | ----------- | ------------------------------------------------------------ |
+| 80          | `TCP`       | HTTP port used to serve applications through the OpenShift router. |
+| 443         | `TCP`       | HTTPS port used to serve applications through the OpenShift router. |
+| 6443        | `TCP`       | HTTPS API port for the MicroShift API |
+| 5353        | `UDP`       | mDNS service to respond for OpenShift route mDNS hosts |
```

Comment:
... *if* using mDNS. Maybe with link to the mDNS page.

Reply:
+1